src/helpers/utils.ts

```diff
@@ -60,7 +60,13 @@ export interface ProxyConfiguration {
   protocol: ProxyType
 }
 
-export const getRequestBuilder = (baseUrl: string, apiKey: string, appKey?: string, proxyOpts?: ProxyConfiguration) => {
+export const getRequestBuilder = (
+  baseUrl: string,
+  apiKey: string,
+  appKey?: string,
+  proxyOpts?: ProxyConfiguration,
+  disableEnvironmentVariables = false
+) => {
```

Comment:
nit: maybe we should use an object here instead of a growing list of parameters.

Reply:
Doing it in another PR to ease reviews of other teams.